project/src/components/property-screen/review.tsx

```diff
@@ -0,0 +1,40 @@
+import { ReviewData } from '../../types/offers-type';
+
+type ReviewProps = {
+  review: ReviewData;
```

Comment:
Можно упростить и не потеряется смысл

```suggestion
  review: Review;
```